src/pixi-multistyle-text.ts

```diff
@@ -2,48 +2,96 @@
 
 "use strict";
 
-interface TextStyle {
-  align?: string;
-  breakWords?: boolean;
-  dropShadow?: boolean;
-  dropShadowAlpha?: number;
-  dropShadowAngle?: number;
-  dropShadowBlur?: number;
-  dropShadowColor?: string | number;
-  dropShadowDistance?: number;
-  fill?: string | string[] | number | number[] | CanvasGradient | CanvasPattern;
-  fillGradientType?: number;
-  fillGradientStops?: number[];
-  fontFamily?: string | string[];
-  fontSize?: number | string;
-  fontStyle?: string;
-  fontVariant?: string;
-  fontWeight?: string;
-  leading?: number;
-  letterSpacing?: number;
-  lineHeight?: number;
-  lineJoin?: string;
-  miterLimit?: number;
-  padding?: number;
-  stroke?: string | number;
-  strokeThickness?: number;
-  trim?: boolean;
-  textBaseline?: string;
-  whiteSpace?: string;
-  wordWrap?: boolean;
-  wordWrapWidth?: number;
+type StringMap = { [key: string]: string };
```

Comment:
nit: Not sure a `StringMap` type is really useful, you can use `Record<string, string>` for this

Reply:
OK. just trying to reduce the duplication of that type through the file. but Record works too.